react/features/participants-pane/components/native/MeetingParticipantItem.js

```diff
@@ -178,6 +179,10 @@ function mapStateToProps(state, ownProps): Object {
     const audioMediaState = getParticipantAudioMediaState(participant, _isAudioMuted, state);
     const videoMediaState = getParticipantVideoMediaState(participant, _isVideoMuted, state);
     const { disableModeratorIndicator } = state['features/base/config'];
+    const raisedHand = hasRaisedHand(participant?.local
```

Comment:
Why don't we just pass the `participant` object here?

Reply:
Because it doesn't get the updated raised hand timestamp. Basically raise / lower hand doesn't cause a re-render so we can't see any changes live

Reply:
👍